cmd/up/up.go

```diff
@@ -957,6 +965,8 @@ func (up *upContext) shutdown() {
 	}
 
 	log.Info("completed shutdown sequence")
+	up.Canceled <- true
```

Comment:
can we rename this to up.ShutdownCompleted (or somethign like that), since this could happen on a regular shutdown as well, and not just when canceling?

Reply:
ok